README.md

```diff
@@ -1,77 +1,31 @@
-focused on Heat Seek.
-
-The goals are;
+The goals of this repo are:
 
 * Create a dataset comprised of various NYC housing data that will can be used
 as the source data to identify and predict bad landlords.
 * Provide a way to create reports suitable for sending to lawyers and activists.
 * Expose this data via a web app.
 
+## Getting Started
 
-Python Notebooks
-***************************
-
-As we are committing full Python notebooks, be sure to clear output before
-saving. (`Cell` -> `All Output` -> `Clear`). This will help us keep diffs clean
-as we work.
-
-
-Getting Started
-***************************
-
-There are two options for getting started with this repo.
-
-The first is to install MySQL, and Python packages listed in requirements.txt.
-Ditto for the rails app. Additionally, MySQL requires additional
-[PRCE UDFs](https://github.com/mysqludf/lib_mysqludf_preg).
-
-The second is to use docker. This method will use an .env file to configure the
-containers.
-
-* [Install Docker and docker-compose](https://docs.docker.com/engine/getstarted/)
-* Run `cp .env.example .env`, and fill in any relevant values
-* Run `docker-compose up`
-* The Jupyter notebook will be running at [localhost:8888](http://localhost:8888)
-* The Rails app will be running at [localhost:3000](http://localhost:3000)
-
-To get into the web container, and run rake tasks, etc;
-
-```bash
-docker-compose exec web /bin/bash
-```
+The recommended way to install and run this repo is using Docker.
 
-To examine MySQL directly you can use docker-compose to exec the mysql client;
+The simplest way to install Docker is to install [the Platform from the official website](https://www.docker.com/products/docker). The official website offers [a good tutorial](https://docs.docker.com/engine/getstarted/).
 
-```bash
-docker-compose exec db mysql -u deltanyc -ppassword deltanyc
-```
+Before launching any of our containers, you'll need to create a .env file. The required fields are listed in .env.example. The defaults are likely good enough for running an instance of the site locally, but you'll need to obtain [an API key for the NYC geoclient](https://developer.cityofnewyork.us/api/geoclient-api) for some of the data cleaning steps to function.
 
-OR you can connect to container over 3306 using the local mysql client if you
-have it installed. See [this article](http://stackoverflow.com/a/32361238/103315)
-on why you need to use `127.0.0.1`.
+`docker-compose up` will install and launch all our containers.  These containers are:
 
-```bash
-mysql -h 127.0.0.1 -P 3306 -u deltanyc -ppassword deltanyc
-```
+* db: The MySQL database storing NYC city data both in semi-raw form and joined into a model for the Rails website. The database can be examined with `docker exec deltanycheatseek_db_1 mysql -u <user from .env> -p <password from .env>`. Local clients can connect at 127.0.0.1:3306 ([localhost will not work](http://stackoverflow.com/a/32361238/103315)).
```

Comment:
That's what you originally had written in the README, but I've never been able to get it to work.
For example, `docker exec db /bin/bash` (or even with deltanycheatseek_db) gets:
"Error response from daemon: No such container: db".